### ISSUE TITLE
icon-click does not trigger on keyboard action

### ISSUE THREAD

**Issue by mloeffle** [formkit/formkit#1479]
### Reproduction

https://formkit.com/playground?fkv=latest&fileTab=Playground.vue&css-framework=genesis&files=jc%5B%28%27name%21%27Playground.vue%27%7Eeditj%21%27%3CscripNsetupDQsNG7+%7Bnode%2C+e%7D7%3E+%282+HHUWXLYW%222+3V73V7UZXtextYZ%222%292JscriptD2%3CM8%3CFjmKit288V4Z1label4A+fancy+Z+input1value4mySecretP6%27%3A1preB4Z1RB4L1%40_ick4G1_ass4hover%3Atext-blue-500O28%2FDJM2%3CstyleqcopedD%2F*2This+filequppjts+Tailwind+CSS.+22If+youzeed+vanilla+CSS+iNcan+go+here.2*%2F2JstyleD%27%7Eadded%21true%29%5D1O2882%5Cn3zode.props.4%3DO6asswjd7+%3D8++9+%22Bfix-iQD%3E2E%22+GhandleIQClickH3RfixIQ7J%3C%2FLWClosedMtemplateDNt+O%5C%27QconRsufU%3D%3D9VtypeWeyeXE%3F9YE%3A9Zp6_RB-cljorq+sz+n%01zqj_ZYXWVURQONMLJHGEDB98764321_

### Describe the bug

Hi there,

When testing how my implementation works accessibility wise i realised icon click handlers only work for actual mouse click events. You cannot use only the keyboard to interact with everything.
I could not find anything in the docs about this in particular, so i assume its a bug.

Expectation:
When there is a associated click handler I can use tab to focus the icon and use enter and/or space to trigger the associated action.

Current:
The Icon can be focused but the action cannot be triggered by a keyboard event.
This behaviour can be seen in the doc about [icon click handlers](https://formkit.com/essentials/icons#icon-click-handlers) which uses an example to toggle the input type between text and password. Click works, tabbing and using enter or space does not.
The behaviour occours when using `suffix/prefix-icon-click` attribute as well as using the `FormKitIcon` component with an `@click` action

### Environment

• OS: iOS
• Browser chrome, safari, firefox
• Version

**Comment by justin-schroeder** [comment]
Opened PR: https://github.com/formkit/formkit/pull/1744

Local verification:
- `pnpm build icons` (local package entry prerequisite for the Vue FormKit spec)
- `pnpm vitest packages/vue/__tests__/FormKitIcon.spec.ts packages/vue/__tests__/FormKit.spec.ts packages/react/__tests__/FormKitIcon.spec.ts packages/react/__tests__/inputs/checkbox.spec.ts --run`
- `pnpm build inputs`
- `pnpm build themes`
- `pnpm build vue`
- `pnpm build react`
- `git diff --check`

Security note: this only maps Enter/Space keyboard activation to the existing click behavior for already-clickable icons; it adds no dependencies, network calls, or user-controlled HTML execution paths.

**Comment by justin-schroeder** [comment]
This should be fixed in 2.1.0, which was just released. Please upgrade your `@formkit/*` packages to `^2.1.0` and let us know if you're still seeing the issue. Thanks!
